addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CrossSiteScriptingScanRule.java

```diff
@@ -677,6 +678,81 @@ private boolean performImageTagAttack(HtmlContext context, HttpMessage msg, Stri
         return false;
     }
 
+    private boolean performAttributeAttack(HtmlContext context, HttpMessage msg, String param) {
+        String attackString = "accesskey=\"x\" onclick=\"alert(1)\" b";
```

Comment:
I think its worth extracting the `"accesskey=\"x\" onclick=\"alert(1)\"` string into a static at the top of the file, esp as its used twice (so far).
Its also worth defining another one using single quotes and trying both - some apps may block one or other of the quotes.
It would also be good to add "other" info explaining what the "accesskey" is for - that will help people trying to reproduce the issue.